qemu/include/elf.h

```diff
@@ -1,9 +1,294 @@
 #ifndef _QEMU_ELF_H
 #define _QEMU_ELF_H
 
+#include "platform.h"
```

Comment:
yes, you can see the current file at https://github.com/unicorn-engine/unicorn/blob/master/qemu/include/elf.h